site/_quarto.yml

```diff
@@ -250,7 +250,7 @@ format:
     mainfont: Inter
     css: styles.css
     link-external-newwindow: true
-#    link-external-icon: true
+    link-external-icon: true
```

Comment:
I'd disabled this yesterday when I was experimenting with links back to validmind.ai. This change is correct.